### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by unindented**
- Bumped version of `react-styleguidist` and `react-styleguidist-visual`.
- Removed stale screenshots, as `react-styleguidist-visual` now errors if there are any.

### CODE REVIEW

File: config/styleguide-visual/config.js [hunk 1]
```diff
@@ -15,6 +15,6 @@ module.exports = {
     args: ['--no-sandbox', '--disable-setuid-sandbox'],
   },
   navigationOptions: {
-    waitUntil: 'networkidle',
+    waitUntil: 'networkidle0',
```
Comment (unindented) [inline config/styleguide-visual/config.js]: `puppetteer v1.0.0` renamed this.

**Comment by swese44** [comment]
why did visual diffs change height? also i think the newer version of styleguidist fixed some routing issues, are we able to speed up our diffs by not reloading each page?

**Comment by unindented** [comment]
@swese44 I think font rendering has changed with the latest version of Chrome, at least on Linux. You can check with the swipe tool.

And yes, I thought they had fixed routing, but it's broken again. Try going to https://react-styleguidist.js.org/examples/basic/, clicking on an example and going back, and some examples will disappear.

**Comment by adrianmg** [comment]
Everything looks correct, the new rendering is slightly more accurate and similar to how Sketch is rendering Segoe. I wonder if this is coming from a change in the rendering engine or from actually an update of the font-family we're using. Some of the changes are not just small rendering differences but actually tweaking descendants and other things.

Everything looks correct since it's either expanding up or down but keeping all the baselines in the same place.

**Pull request merged**